src/units/attack_type.hpp

```diff
@@ -365,6 +417,13 @@ class attack_type : public std::enable_shared_from_this<attack_type>
 	int parry_;
 	config specials_;
 	bool changed_;
+	mutable config check_special_;
```

Comment:
Things need comments, and names that will make sense when looking at the code in six months' time, when working on a different bug.

Reply:
removed this variable